layouts/partials/head.html

```diff
@@ -0,0 +1,36 @@
+<head>
+  {{- partial "header-scripts.html" . -}}
+
+  <meta charset="utf-8">
+  {{- partial "prefetch.html" . -}}
+  {{- partial "preload.html" . -}}
+
+  <title>
+    {{/* if isset .Params "integration_title" }}
+      {{ .Params.integration_title }}
+    {{ else }}
+      {{ .Title }}
+    {{ end */}}
+
+    {{- cond (isset .Params "integration_title") .Params.integration_title .Title -}}
+  </title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  {{- if .Params.external_redirect -}} 
+    {{- partial "meta-http-equiv.html" . -}} 
+  {{- end -}}
+
+  {{- partial "canonical.html" . -}}
+  {{- partial "noindex.html" . -}}
+  {{- partial "hreflang.html" . -}}
+
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <link rel="icon" type="image/png" href="https://docs.datadoghq.com/favicon.ico">
+
+  {{- partialCached "css.html" . -}}
+
+  {{- if ne $.Params.disable_opengraph_meta_tags true -}}
+    {{- partial "meta.html" . -}}
+  {{- end -}}
+</head>
```

Comment:
I believe we are going to need to bring back `<script src="{{ (index $.Site.Data.manifest "dd-browser-logs-rum.js") | relURL }}"></script>` somehow in here. It needs to be separate or we risk redeploying every html file every build.

Reply:
@davidejones the `dd-browser-logs-rum.js` JS file is now being loaded in separately from the main bundle, without fingerprinting:  https://github.com/DataDog/documentation/blob/brian.deutsch/hugo-js-pipes/layouts/partials/javascript.html#L21

Reply:
Thanks! this works at least until we can figure out a better solution